stdlib/public/core/Arrays.swift.gyb

```diff
@@ -151,7 +151,7 @@ if True:
 /// Here's an implementation of those steps:
 ///
 ///     if let i = absences.index(where: { $0 > 0 }) {                      // 1
-///         let absencesAfterFirst = absences.suffix(from: i + 1)           // 2
+///         let absencesAfterFirst = absences[(i + 1)...]                   // 2
```

Comment:
`absences[i+1...]` will work without the extra parens.  I personally think cases like this read better without spaces around the `+`, but you can always do `absences[i + 1 ...]` if you must have them.